easypost/__init__.py

```diff
@@ -272,8 +273,10 @@ def request_raw(self, method, url, params=None, apiKeyRequired=True):
                 'at contact@easypost.com for assistance.')
 
         abs_url = self.api_url(url)
+        if (url == "/rating/v1/rates"):
```

Comment:
To me, I would have not use the `isRating` var on line 264/278, and would have done this `abs_url` re-assignment down on line 303 where the header is being modified, so that this is all in one place. However, the url assignment would now be happening far away from the original `abs_url` assignment, which I don't like. @DorianGray thoughts?

(don't make this change unless someone else weighs in)